Add open graph

clementbiron opened the pull request:
Adds open graph data, documents how to manage images of this type, adds a default image and the image for the last Telegram memo.

MattiSG commented on CONTRIBUTING.md:
```diff
@@ -37,6 +37,10 @@ It is strongly recommended to use an image compression tool such as [ImageOptim]
 
 If you need to use different illustrations depending on the language of the site, you can do so by adding a suffix to the name of your image file. Supported suffix values are `en` and `fr` and it must be preceded by a `-`. For example: `research-on-the-impact-on-human-rights-fr.png`.
 
+## Adding an Open Graph image
```
```suggestion
## Adding an Open Graph image (social media preview image)
```

MattiSG commented on CONTRIBUTING.md:
```diff
@@ -37,6 +37,10 @@ It is strongly recommended to use an image compression tool such as [ImageOptim]
 
+## Adding an Open Graph image
+
+Add an 1200*630px image in JPG format to the [`assets/images/opengraph/<language_tag>/` folder](https://github.com/OpenTermsArchive/opentermsarchive.org/tree/master/assets/images/opengraph/) where <language_tag> is the language tag as defined by [RFC 5646](https://datatracker.ietf.org/doc/html/rfc5646) and the file is named with [sanitized](https://gohugo.io/functions/urls/urlize/) [title](https: ...
```
What should 
```suggestion
Add an 1200*630px image in JPG format to the [`assets/images/opengraph/<language_tag>/` folder](https://github.com/OpenTermsArchive/opentermsarchive.org/tree/master/assets/images/opengraph/) where <language_tag> is the language tag as defined by [RFC 5646](https://datatracker.ietf.org/doc/html/rfc5646) and the file is named with the [sanitized](https://gohugo.io/functions/urls/urlize/) [title](https://gohugo.io/methods/page/title/) of the page for which you want to configure the Open Graph image. The extension must be `.jpg` (not `.jpeg`, all lowercase).
```

MattiSG commented on themes/opentermsarchive/layouts/partials/head.html:
```diff
@@ -17,4 +17,14 @@
+    {{ $openGraphImage = . }}
+  {{ end }}
+  <meta property="og:image" content="{{ $openGraphImage.Permalink }}">
+  <meta property="og:image:width" content="1200">
+  <meta property="og:image:height" content="630">
```
Why not use the exact image width and height? 🙂
https://gohugo.io/content-management/image-processing/#image-rendering

MattiSG reviewed (commented):
Nice!

I believe most illustrations will be screenshots or diff, and PNG is much more efficient than JPG for such image types. I would thus suggest to use PNG instead of JPG as a default. This would also enable using the Open Graph image to illustrate the article in #350, preventing assets duplication 🙂

Ndpnt commented:
X and Linkedin seem to not have description https://www.opengraph.xyz/url/https%3A%2F%2Fdeploy-preview-351--open-terms-archive-website.netlify.app%2Ffr%2F

clementbiron commented:
Thanks @Ndpnt, i've added open graph description meta tag and updated the meta name description to automatically generate a content if no `html_description` param is provided.

LinkedIn is supposed to [support](https://www.linkedin.com/help/linkedin/answer/a521928/making-your-website-shareable-on-linkedin?lang=en) `og:description` and X to [fallback](https://developer.x.com/en/docs/x-for-websites/cards/guides/getting-started) to Open Graph property if Twitter-specific property is not present.
